framework/src/org/checkerframework/framework/util/typeinference/DefaultTypeArgumentInference.java

```diff
@@ -113,14 +111,14 @@
     public void adaptMethodType(AnnotatedTypeFactory typeFactory, ExpressionTree invocation, AnnotatedExecutableType methodType) {
         //do nothing
     }
-
+    // TODO: THIS IS A BIG VIOLATION OF Single Responsibility and SHOULD BE FIXED, IT IS SOLELY HERE
```

Comment:
You moved this comment from below getArgumentTypes to inbetween adaptMethodType and getArgumentTypes - with no spacing.
To which of the two methods should it apply?
Is the whole method a kludge or something happening within it?
Which part is the single responsible party where the method should be moved to?